src/pipelines/infra.yaml

```diff
@@ -15,6 +15,7 @@ variables:
   - name: tag
     value: "$(Build.BuildId)"
   - group: TFBackend
+  - group: ACIDevOps.Secrets
   - name: skipComponentGovernanceDetection
```

Comment:
This is not used anywhere. I'm guessing it was mistakenly copied over from the engagement.

Reply:
@cesar-younes Actually it is used. All the variables related to Azure DevOps in the TF command line `-var devops_org_name=$(devopsOrganization) -var devops_personal_access_token=$(devopsPersonalAccessToken) -var devops_pool_name=$(devopsPoolName) -var devops_agent_docker_image=$(devopsAgentDockerImage) -var devops_agent_docker_tag=$(devopsAgentDockerTag)"` are coming from this group.